Sail/src/Sail/entities/components/MapComponent.h

```diff
@@ -92,7 +92,7 @@ class MapComponent : public Component<MapComponent> {
 #endif
 	int totalArea = xsize * ysize;
 	int numberOfRooms = 1;
-	float tileSize = 7.f;
+	const static int tileSize = 7;
```

Comment:
Why has this been changed to a const int?

Reply:
It is changed to static because I need to access it without having access to an instance of  the class. xsize and ysize is done the same way a few lines above.